### PR TITLE
Support create_ports and fix bug in generated code

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/saiserver/ports_test.go [hunk 1]
```diff
@@ -195,6 +195,109 @@ func TestCreatePort(t *testing.T) {
...
+			}
+			if d := cmp.Diff(got, tt.want, protocmp.Transform()); d != "" {
+				t.Errorf("CreatePort() failed: diff(-got,+want)\n:%s", d)
+			}
+			attr := &saipb.PortAttribute{}
+			if err := mgr.PopulateAllAttributes("1", attr); err != nil {
+				t.Fatal(err)
+			}
+			if d := cmp.Diff(attr, tt.wantAttr, protocmp.Transform()); d != "" {
+				t.Errorf("CreatePort() failed: diff(-got,+want)\n:%s", d)
```
Comment (guoshiuan) [inline dataplane/saiserver/ports_test.go:295]: CreatePorts

File: dataplane/saiserver/ports_test.go [hunk 2]
```diff
@@ -195,6 +195,109 @@ func TestCreatePort(t *testing.T) {
...
+			NumberOfIngressPriorityGroups:    proto.Uint32(0),
+			QosMaximumHeadroomSize:           proto.Uint32(0),
+			AdminState:                       proto.Bool(true),
+			AutoNegMode:                      proto.Bool(true),
+			Mtu:                              proto.Uint32(1514),
+		},
+	}}
+	for _, tt := range tests {
+		t.Run(tt.desc, func(t *testing.T) {
+			getInterface = func(name string) (*net.Interface, error) {
```
Comment (guoshiuan) [inline dataplane/saiserver/ports_test.go:274]: Where is this func used?

File: dataplane/saiserver/ports_test.go [hunk 3]
```diff
@@ -195,6 +195,109 @@ func TestCreatePort(t *testing.T) {
...
+		getInterfaceErr error
+		want            *saipb.CreatePortsResponse
+		wantAttr        *saipb.PortAttribute
+		wantErr         string
+	}{{
+		desc: "success",
+		req: &saipb.CreatePortsRequest{
+			Reqs: []*saipb.CreatePortRequest{{}},
+		},
+		getInterfaceErr: fmt.Errorf("no interface"),
```
Comment (guoshiuan) [inline dataplane/saiserver/ports_test.go:211]: Why does a successful TC need an interface error?

File: dataplane/standalone/sai/fdb.cc [hunk 4]
```diff
@@ -233,6 +233,9 @@ sai_status_t l_create_fdb_entries(uint32_t object_count,
     LOG(ERROR) << status.error_message();
     return SAI_STATUS_FAILURE;
   }
+  if (object_count != resp.resps().size()) {
```
Comment (guoshiuan) [inline dataplane/standalone/sai/fdb.cc:236]: Are the following .cc files code auto-generated?

File: dataplane/apigen/ccgen/ccgen.go [hunk 5]
```diff
@@ -668,8 +668,11 @@ return msg;
 		LOG(ERROR) << status.error_message();
 		return SAI_STATUS_FAILURE;
 	}
+	if (object_count != resp.resps().size()) {
```
Comment (guoshiuan) [inline dataplane/apigen/ccgen/ccgen.go:671]: Wonder what can cause this situation to happen?

Comment (DanG100) [reply dataplane/apigen/ccgen/ccgen.go:671]: if there's a bug in the server and it doesn't return the correct number of objects.

File: dataplane/saiserver/ports_test.go (hunk 3 above)
Comment (DanG100) [reply dataplane/saiserver/ports_test.go:211]: good catch, too much copy paste

File: dataplane/saiserver/ports_test.go (hunk 2 above)
Comment (DanG100) [reply dataplane/saiserver/ports_test.go:274]: same here, removed

File: dataplane/saiserver/ports_test.go (hunk 1 above)
Comment (DanG100) [reply dataplane/saiserver/ports_test.go:295]: fixed

File: dataplane/standalone/sai/fdb.cc (hunk 4 above)
Comment (DanG100) [reply dataplane/standalone/sai/fdb.cc:236]: yes

**Comment by DanG100** [comment]
the integration tests should pass after #354 is merged

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**